<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogvehicule</class>
 <widget class="QDialog" name="dialogvehicule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Vehicule</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="button_music">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>30</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Musique</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>590</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Main Menu</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>791</width>
     <height>501</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID Vehicule</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Destination</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID Employer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_vehicule">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>561</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="destination">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>561</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_employer">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>561</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_ajouter">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>260</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_stock">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>561</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID Stock</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Afficher</string>
    </attribute>
    <widget class="QTableView" name="tabvehicule">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>781</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="tri1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>251</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Changer Tri Desc (ID 1)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tri2_1">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>410</y>
       <width>261</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Changer Tri Desc(Stock)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tri1_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>251</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Changer Tri Asc (ID 1)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tri2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>410</y>
       <width>261</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Changer Tri Asc(Stock)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tri3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>251</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Changer Tri Asc(Destination)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tri3_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>251</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Changer Tri Desc(Destination)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tri6">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>350</y>
       <width>261</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Changer Tri Desc(Employer)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="tri6_1">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>350</y>
       <width>261</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Changer Tri Asc(Employer)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="rech">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>1</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Supprimer</string>
    </attribute>
    <widget class="QLineEdit" name="vehicule_sup">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>581</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Id Vehicule</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Modifier</string>
    </attribute>
    <widget class="QLineEdit" name="nouvid_emp">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>190</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_save">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>240</y>
       <width>93</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Sauvgarder</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="idveh_mod">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="evall">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>230</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nouveau ID Stock</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_edit">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>60</y>
       <width>93</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nouvid_stock">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>230</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="four2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nouveau ID Employer:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID Vehicule</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nouv_destin">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>140</y>
       <width>231</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="four2_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nouveau Destination</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
